core/src/integration-test/java/com/scalar/db/storage/StorageWithReservedKeywordIntegrationTestBase.java

```diff
@@ -0,0 +1,531 @@
+package com.scalar.db.storage;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import com.scalar.db.api.ConditionalExpression;
+import com.scalar.db.api.Delete;
+import com.scalar.db.api.DeleteIf;
+import com.scalar.db.api.DeleteIfExists;
+import com.scalar.db.api.DistributedStorage;
+import com.scalar.db.api.DistributedStorageAdmin;
+import com.scalar.db.api.Get;
+import com.scalar.db.api.Put;
+import com.scalar.db.api.PutIf;
+import com.scalar.db.api.PutIfNotExists;
+import com.scalar.db.api.Result;
+import com.scalar.db.api.Scan;
+import com.scalar.db.api.Scanner;
+import com.scalar.db.api.TableMetadata;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.exception.storage.ExecutionException;
+import com.scalar.db.io.BooleanValue;
+import com.scalar.db.io.DataType;
+import com.scalar.db.io.IntValue;
+import com.scalar.db.io.Key;
+import com.scalar.db.io.TextValue;
+import com.scalar.db.service.StorageFactory;
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.IntStream;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+@SuppressFBWarnings({"ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD", "MS_CANNOT_BE_FINAL"})
+public abstract class StorageWithReservedKeywordIntegrationTestBase {
+
+  protected static String NAMESPACE = "integration_testing";
+  protected static String TABLE = "test_table";
+  protected static String COL_NAME1 = "c1";
+  protected static String COL_NAME2 = "c2";
+  protected static String COL_NAME3 = "c3";
+  protected static String COL_NAME4 = "c4";
+  protected static String COL_NAME5 = "c5";
```

Comment:
It looks like these don't need to be `static`. And if these namespace and table and column names are different for each adaptor, I think we can add abstract methods as follows and implement them for each adaptor's test:
```java
  protected abstract String getNamespace();

  protected abstract String getTable();

  protected abstract String getColumnName1();

  protected abstract String getColumnName2();

  protected abstract String getColumnName3();

  protected abstract String getColumnName4();

  protected abstract String getColumnName5();
```

Reply:
Thanks for the suggestion. I changed to use abstract methods [commit](https://github.com/scalar-labs/scalardb/pull/441/commits/0216d485c07c6ab2c4576fb11edd1ce99fced0b6).
I'm not sure but when don't use `static` the column name values become null after running the first test, so I kept using `static` for them.